plugin.video.nhlgcl/resources/lib/globals.py

```diff
@@ -1,28 +1,31 @@
 import sys
 import re, os, time
-import calendar
-import pytz
-import urllib
-import requests
-import json
-import http
-import time
+import calendar, pytz
+import requests, urllib
 from bs4 import BeautifulSoup
-from datetime import date, datetime, timedelta
-import xbmc, xbmcvfs, xbmcplugin, xbmcgui, xbmcaddon
+from datetime import datetime, timedelta
+from kodi_six import xbmc, xbmcvfs, xbmcplugin, xbmcgui, xbmcaddon
 
+if sys.version_info[0] > 2:
+    import http
+    cookielib = http.cookiejar
+    urllib = urllib.parse
+else:
+    import cookielib
 
-urllib = urllib.parse
-cookielib = http.cookiejar
+try:
+    xbmc.translatePath = xbmcvfs.translatePath
+except AttributeError:
+    pass
 
 addon_handle = int(sys.argv[1])
 
 # Addon Info
 ADDON = xbmcaddon.Addon()
 ADDON_ID = ADDON.getAddonInfo('id')
 ADDON_VERSION = ADDON.getAddonInfo('version')
-ADDON_PATH_PROFILE = xbmcvfs.translatePath(ADDON.getAddonInfo('profile'))
-XBMC_VERSION = float(re.findall(r'\d{2}\.\d{1}', xbmc.getInfoLabel("System.BuildVersion"))[0])
+ADDON_PATH_PROFILE = xbmc.translatePath(ADDON.getAddonInfo('profile'))
```

Comment:
sorry, you have to use xbmcvfs.translatepath for matrix++
xbmc.translatepath will be removed in v20

Reply:
I think line 16 through 19 should do the conversion for xbmcvfs

Reply:
You're right